workflow-service/src/main/java/com/redhat/parodos/project/repository/ProjectRepository.java

```diff
@@ -26,4 +29,8 @@
  */
 public interface ProjectRepository extends JpaRepository<Project, UUID> {
 
+	List<Project> findAllByUserUsername(String username);
```

Comment:
pls add tests.
should be fairly simple with https://github.com/parodos-dev/parodos/pull/295/files#diff-43f3718db35688f81c93147df5fad5e6caf4bbdc988a24d007edd32db7a89372 (once it is being merged)

Reply:
yes Thank you for the example ~